src/examples/02_animations/parametric_function.py

```diff
@@ -0,0 +1,15 @@
+from manimlib.imports import *
+
+
+def i_need_a_name(t):
```

Comment:
What exactly does this function, is there a mathematical term for what happens here?